erpnext/stock/doctype/delivery_trip/delivery_trip.py

```diff
@@ -157,9 +154,10 @@ def form_route_list(self, optimize):
 		Returns:
 			(list of list of str): List of address routes split at locks, if optimize is `True`
 		"""
+		if not self.driver_address:
```

Comment:
Check on the front-end itself, and don't call `process_route` if address is missing.